daemon/snapshotter/mount.go

```diff
@@ -110,6 +113,23 @@ func (m *refCountMounter) Unmount(target string) error {
 	return nil
 }
 
+func (m *refCountMounter) Mounted(containerID string) (string, error) {
+	mounted, err := m.base.Mounted(containerID)
+	if err != nil || mounted == "" {
+		return mounted, err
+	}
+
+	target := m.base.target(containerID)
+
+	// Check if the refcount is non-zero.
+	m.rc.Increment(target)
+	if m.rc.Decrement(target) > 0 {
```

Comment:
Not sure if I understand this part; we're incrementing , and immediately after that decrementing.

Is this because we don't have a different way to get the active counter?

Reply:
Yeah, `graphdriver.RefCount` only has `Increment` and `Decrement`

Reply:
I created a ticket for this; if we want to track that in code, we _could_ add `TODO` comment here so that we don't forget.

- https://github.com/moby/moby/issues/46841

Reply:
Also opened a tracking issue for handling reference-counting, as I was looking at this code, and stumbled on the hard-coded `` list again 😂 

- opened https://github.com/moby/moby/issues/46842